packages/frontend/features/swap-tokens/SwapOptions.tsx

```diff
@@ -42,6 +49,43 @@ const SwapOptions = () => {
     setAmountToSwap(canSwapAmount.toString());
   };
 
+  const getAmountToReceive = async () => {
+    setAmountToReceive('....')
```

Comment:
This should probably be `null`, and then in the UI it would detect `null` and render `....`. We want to keep the types in our state consistent.